mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -195,9 +236,9 @@ def _azure_upload_file(self, credentials, local_file, artifact_path):
                         "Failed to authorize request, possibly due to credential expiration."
                         " Refreshing credentials and trying again..."
                     )
-                    credentials = self._get_write_credentials(
-                        self.run_id, artifact_path
-                    ).credentials
+                    credentials = self._get_write_credential_infos(
```

Comment:
credentials -> credential or credential_info

Reply:
Fixed the naming here and in the block above. Thanks!